cmd/pimo/main.go

```diff
@@ -155,6 +157,9 @@ func run() {
 	if repeatUntil != "" {
 		source = model.NewTempSource(source)
 	}
+	if repeatWhile != "" {
```

Comment:
If with same content that the if above, so it can be replaced by a single if :

```go
if repeatUntil != "" || repeatWhile != "" {
		source = model.NewTempSource(source)
}
```

Also it might be better to add another test to check if both flags are used at the same time

```go
if repeatUntil != "" && repeatWhile != "" {
		log.Err(err).Msg("Cannot use repeatUntil and repeatWhile flags together")
		log.Warn().Int("return", 1).Msg("End PIMO")
		os.Exit(1)
}
```